src/runtime/local/vectorized/Tasks.cpp

```diff
@@ -65,6 +65,11 @@ void CompiledPipelineTask<DenseMatrix<VT>>::execute(uint32_t fid, uint32_t batch
     }
 }
 
+template<typename VT>
+uint64_t CompiledPipelineTask<DenseMatrix<VT>>::getTaskSize() {
+return _data._ru;
```

Comment:
Just checking: Why is this different from the CSR version below?

Reply:
OK thx for the update. I'll have it fixed with a few other things to make the CUDA code paths fly again.